### PR TITLE
fixed bundles issues

### PR DESCRIPTION
**Pull request by devgurjar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/pom.xml [hunk 1]
```diff
@@ -86,9 +86,15 @@
...
                         <_metatypeannotations>*</_metatypeannotations>
                         <Import-Package>
-                            javax.annotation;version=0.0.0,
+<!--                            javax.annotation;version=0.0.0,-->
+                            <!-- https://github.com/jhy/jsoup/issues/1616 -->
+                            javax.annotation;version=!;resolution:=optional,javax.annotation.meta;version=!;resolution:=optional,
                             *
                         </Import-Package>
+                        <Embed-Dependency>
+                            core.wcm.components.core;
```
Comment (nit23uec) [inline bundles/af-core/pom.xml:95]: why jsoup is embedded?

Comment (devgurjar) [reply bundles/af-core/pom.xml:95]: it is coming from core.wcm.components.core;

Comment (devgurjar) [reply bundles/af-core/pom.xml:95]: com.adobe.aem:core-forms-components-examples-core:bundle:1.1.7-SNAPSHOT
[INFO] +- com.adobe.cq:core.wcm.components.core:jar:2.21.2:provided
[INFO] |  +- org.apache.sling:org.apache.sling.servlets.annotations:jar:1.2.4:provided
[INFO] |  \- org.jsoup:jsoup:jar:1.14.3:provided
[INFO] +- org.jetbrains:annotations:jar:17.0.0:provided
[INFO] +- uk.org.lidalia:slf4j-test:jar:1.2.0:test
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.6:provided

Comment (devgurjar) [reply bundles/af-core/pom.xml:95]: @nit23uec I have tested the newly created env and DoR is working fine

Comment (nit23uec) [reply bundles/af-core/pom.xml:95]: ideally, we wcm core components should bring their own jsoup. I'm not blocking the merge due to this. But this is undesired.

**Pull request merged**